docker/test/base/setup_export_logs.sh

```diff
@@ -5,61 +5,150 @@
 # and their names will contain a hash of the table structure,
```

Comment:
Interesting, why the code start to be so much more complex?

Reply:
Arguable. There was plenty of the same code floating back and forth from Python to bash. Now there's a single place to get credentials and arguments for the docker container and a single place to set up the logs pushing.

Besides that, previously the code could carelessly print the credentials to the stdout because actions would care about secrets. Now we need to care for them on our own, and not pass them as ENV parameters. But that's the concern I've raised from the very beginning.

It's a tiny particle of questions we'll need to address in case of moving somewhere else from GH actions. But I'd like to address such moments as soon as possible in any case, to at least not stuck into them even deeper.